platform/default/sqlite3.hpp

```diff
@@ -42,27 +48,23 @@ class Database {
 
     explicit operator bool() const;
 
-    static void errorLogCallback(void *arg, const int err, const char *msg);
     void setBusyTimeout(std::chrono::milliseconds);
     void exec(const std::string &sql);
     Statement prepare(const char *query);
 
-    int64_t lastInsertRowid() const;
```

Comment:
> for keeping them in Database we'll need a temporary QSqlQuery object that makes the calls based on that db.

This idea won't work as expected - `QSqlQuery::lastInsertId` expects an active QSqlQuery and it wouldn't change the fact that the result would still be potentially overwritten by another statement.

> cache the values in Statement after every execution

This sounds like the way to go - we can cache last insert row ID and the number of rows affected on every `Statement::run` - which can also be done for the default sqlite3 implementation.